index.bs

```diff
@@ -71,8 +74,32 @@ Examples {#examples}
 Security and Privacy Considerations {#security-and-privacy}
 ===================================
 
-There are no specific security and privacy considerations
-beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
+Ambient Light Sensor provides information about lighting conditions near
+the device environment. Potential privacy risks include:
+
+  - Information leaks about the user’s surrounding and work habits.
+  - Profiling. Readout from Ambient Light Sensor can potentially induce
+    information leaks about the user’s interests, web use and work
+    habits, as well as the users’ surrounding. This information might be used
+    to enhance the user profiling and behavioral analysis.
+  - Cross-device linking and tracking. Access to sufficiently precise readouts
+    of Ambient Light Sensor potentially enhance cross-device linking
+    techniques. Such situation may arise if two different devices access web
+    sites including same third-party scripts that compare lighting levels over
+    time.
+  - Cross-device communication. Verbose readout of Ambient Light Sensor could
+    be applied to receive messages emitted by other devices in nearby
+    location. A simple messaging method could arise by multiple devices
+    flashing their screens or camera LEDs and reading out responses with
+    Ambient Light Sensors.
+
+To mitigate these Ambient Light Sensor specific threats, user agents should
+use the following mitigation strategies:
+  - <a>limit maximum sampling frequency</a>, and
```

Comment:
should it be "and or"? At the moment, it sounds that US should use both simultaneously.